multinode-demo/setup.sh

```diff
@@ -86,8 +86,8 @@ if $node_type_leader; then
   echo "Creating $SOLANA_CONFIG_DIR/mint.json with $num_tokens tokens"
   $solana_mint <<<"$num_tokens" > "$SOLANA_CONFIG_PRIVATE_DIR"/mint.json
 
-  echo "Creating $SOLANA_CONFIG_DIR/genesis.log"
-  $solana_genesis < "$SOLANA_CONFIG_PRIVATE_DIR"/mint.json > "$SOLANA_CONFIG_DIR"/genesis.log
+  echo "Creating $SOLANA_CONFIG_DIR/ledger.log"
+  $solana_genesis < "$SOLANA_CONFIG_PRIVATE_DIR"/mint.json > "$SOLANA_CONFIG_DIR"/ledger.log
```

Comment:
That file is the genesis block. Why rename to something less precise?

Reply:
ledger.log will be re-used forever, "genesis" seemed inappropriate.

once the network is up, is separation of the genesis block important?

I can add a step in bash to copy genesis.log into ledger.log, but genesis.log becomes an appendix, a tonsil